solutions/devsprint-firas-safa-1/GitHubApp/Data/Network/Endpoint.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Endpoint.swift
+//  GitHubApp
+//
+//  Created by Razee Hussein-Jamal on 09/02/2022.
+//
+
+import Foundation
+
+struct Endpoint {
+    var host: String
+    var path: String
+    var method: HTTPMethod
+}
+
+extension Endpoint {
+    var url: URL {
+        var components = URLComponents()
+        components.scheme = "https"
+        components.host = host
+        components.path = path
+        
+        guard let url = components.url else { preconditionFailure("Invalid URL components: \(components)")}
```

Comment:
Brackets content on new line

Reply:
In this particular case, since the URL given the conditions can never fail, even with an empty string, it is ok to force unwrap